master/internal/config/config.go

```diff
@@ -108,7 +117,8 @@ func DefaultConfig() *Config {
 				Group: "root",
 			},
 			SSH: SSHConfig{
-				RsaKeySize: 1024,
+				RsaKeySize:   1024,
+				CryptoSystem: RSACryptoSystem,
```

Comment:
I wonder if  @mackrorysd or @rb-determined-ai want to say anything about this?

Reply:
It is faster than reasonably-sized RSA keys.  I never liked the 1024 key size choice when we made it years ago, because key-per-task took too long if you used 2048 keys.  The argument I made was "let's not do one key per task then" And the argument made back to me was "1024 is fine if the tasks are not long lived" and I lost.

Certainly I don't think 1024 rsa keys is a good default by any measure.

I personally would default to ed25519 keys, but I think Ilia is right about what most customers would prefer.

Reply:
Wait, actually, I do vote ed25519 because that's what `ssh-keygen` defaults to.

Reply:
I don't know too much about ssh keys, but it looks like newer algorithms are better in terms of the balance of security and speed, so I put  `ed25519` down as default for now.

Reply:
ed25519 is not in FIPS140, so the category of users who cares about that will have to be aware of it and change the config. ECDSA is still plenty fast and fit most requirements, which feels like the best fit for the default option.

Reply:
Don't have a strong opinion here myself, but to repeat some information added by @dannysauer:

> FWIW: FIPS 186-5, Digital Signature Standard includes ED25519 as of its official publication date Feb last year.
> https://csrc.nist.gov/pubs/fips/186-5/final
> This is helpful, because ECDSA depends on having a good random number generator, which is not always the case.

Reply:
I would side with ED25519 here the more I've read about it. It seems to be the technically superior solution, it is getting accepted in other federal standard, etc.